### PR TITLE
Add documentation for hlt_client

### PR DESCRIPTION
**Pull request by katieamazing**
This PR also revises the starter kit design advice as per Issue #88, clarifies copyright issues for starter kits, and reorganizes the downloads page slightly.

### CODE REVIEW

**Comment by snaar** [comment]
👍

File: website/learn-programming-challenge/contributing.md [hunk 1]
```diff
@@ -63,12 +63,10 @@ Each turn, every bot is sent:
   * A list of this player’s dropoffs.
 3. The changed cells on the game map, represented by an update count (how many cells have changed) and the details of changed cells (x, y, halite amount).
 
-Some objects are recreated from scratch each turn, such as ships. Other objects are reused and updated each turn, such as the the map and player. It’s helpful to be clear when writing your starter kit which objects are recreated and which are reused from turn to turn.
-
 
 <br/>
 ### Submitting your new starter kit
-To submit your starter kit, fork our repo, place your starter package in the `starter-kits/` folder, and send us a pull request! If we accept your PR, your starter package will be added to the site.
+To submit your starter kit, fork our repo, place your starter package in the `starter-kits/` folder, and send us a pull request! Please ensure that your code does not include copyright language; see more in our [Terms of Use](http://localhost:4000/about/terms-and-conditions). If we accept your PR, your starter package will be added to the site.
```
Comment (lidavidm) [inline website/learn-programming-challenge/contributing.md]: Best not to hardcode localhost :)

Comment (lidavidm) [reply website/learn-programming-challenge/contributing.md]: There should be `{{ site.url }}` to insert this for you.

Comment (whatelsie) [reply website/learn-programming-challenge/contributing.md]: Suggestion: 
To submit your starter kit, fork our repo, place your starter package in the `starter-kits/` folder, and send us a pull request! The Halite repo is licensed under the MIT License; please ensure that your code does not include copyright language. If we accept your PR, your starter package will be added to the site.

Comment (katieamazing) [reply website/learn-programming-challenge/contributing.md]: Sounds good!

**Pull request merged**